DataFormats/PatCandidates/interface/MET.h

```diff
@@ -140,28 +149,57 @@ namespace pat {
 
       // ---- members for MET corrections ----
       enum METUncertainty {
-        JetEnUp=0, JetEnDown=1, JetResUp=2, JetResDown=3,
+	JetResUp=0, JetResDown=1, JetEnUp=2, JetEnDown=3,
```

Comment:
I hope nobody relies on int values of these enums: the swap of JetEn and JetRes looks artificial